src/librssguard/core/messagesproxymodel.h

```diff
@@ -27,19 +42,21 @@ class MessagesProxyModel : public QSortFilterProxyModel {
     // Performs sort of items.
     void sort(int column, Qt::SortOrder order = Qt::AscendingOrder);
 
-    bool showUnreadOnly() const;
-    void setShowUnreadOnly(bool show_unread_only);
+    void setFilter(MessageFilter filter);
 
   private:
     QModelIndex getNextImportantItemIndex(int default_row, int max_row) const;
     QModelIndex getNextUnreadItemIndex(int default_row, int max_row) const;
 
     bool lessThan(const QModelIndex& left, const QModelIndex& right) const;
+    bool acceptMessage(Message currentMessage) const;
```

Comment:
Method's name is not self-explanatory enough, perhaps add word "filter" somewhere in method's name.

Reply:
Changed it and thanks